cpp/density/RDF.cc

```diff
@@ -21,6 +21,10 @@ RDF::RDF(unsigned int bins, float r_max, float r_min, bool normalize)
     if (r_max <= 0)
     {
         throw std::invalid_argument("RDF requires r_max to be positive.");
+    }
+        if (r_min < 0)
```

Comment:
Please check the indentation style. I keep requesting the same changes on every review.
```suggestion
    if (r_min < 0)
```